File: src/Python/easy/string/problem2194.py
from typing import List

class Solution:
    def cellsInRange(self, s: str) -> List[str]:
        col1, col2 = s.split(":")
        sheets = []
        for char in range(ord(col1[0]), ord(col2[0]) + 1):
            for i in range(int(col1[1]), int(col2[1]) + 1):
                sheets.append(chr(char) + str(i))
        return sheets


s = "K1:L2"

solution = Solution()
solution.cellsInRange(s)

File: src/JavaScript/easy/string/problem2194.js
/**
 * @param {string} s
 * @return {string[]}
 */

var cellsInRange = function(s) {
    let [col1, col2] = s.split(":")
    let sheets = []
    for(let char = col1[0].charCodeAt(0); char <= col2[0].charCodeAt(0); char++){
        for(let i = parseInt(col1[1]); i <= parseInt(col2[1]); i++){
            sheets.push(String.fromCharCode(char) + i)
        }
    }
    return sheets
};

let s = "K1:L2"
// let s = "A1:F1"

cellsInRange(s)